Library/Formula/tinyproxy.rb

```diff
@@ -11,36 +9,56 @@ class Tinyproxy < Formula
     sha1 "566f27174a811803b447822407dc33f26d306219" => :lion
   end
 
-  option "reverse", "Enable reverse proxying"
+  depends_on "asciidoc" => :build
+
+  option "with-reverse", "Enable reverse proxying"
   option "with-transparent", "Enable transparent proxying"
 
-  depends_on "asciidoc" => :build
+  deprecated_option "reverse" => "with-reverse"
 
   # Fix linking error, via MacPorts: https://trac.macports.org/ticket/27762
   patch :p0 do
     url "https://trac.macports.org/export/83413/trunk/dports/net/tinyproxy/files/patch-configure.diff"
-    sha1 "e946269b681f3ffaa1acd120c93050cb63bfe743"
+    sha256 "414b8ae7d0944fb8d90bef708864c4634ce1576c5f89dd79539bce1f630c9c8d"
   end
 
   def install
+    (var/"log/tinyproxy").mkpath
+    (var/"run/tinyproxy").mkpath
```

Comment:
Do these need to be in `postinstall` to be kept or does specifying the `localstatedir` below mean there's files in them?